### DISCUSSION TITLE
Can I hide or disable today button at actions bar?

### DISCUSSION THREAD

**Discussion by ghost** [palantir/blueprint#5881] [Q&A] [answered]
#### Environment

Version: 3.15.1
Browser: Chrome 79.0.3945.130

#### Question

Hello, I saw a comment saying that 

![image](https://user-images.githubusercontent.com/42964515/73136444-bb47db80-4088-11ea-95d3-3be78d5e4dc0.png)

But I cannot use it properly and cannot find any info regarding this.
Can you help me? I'm trying to hide or disable the today button only.

**Comment by adidahiya** [comment, marked as answer]
Is this for a single date picker or a date range picker?

Yes, you can supply your own `shortcuts` to override the default shortcuts... it should be an array of objects adhering to the `IDatePickerShortcut` interface: https://github.com/palantir/blueprint/blob/bcdb2bceb3bfd96252db07bda4d26fb302f4523b/packages/datetime/src/shortcuts.tsx#L45

if you want to take our default shortcuts and just remove one, you can look at this function for reference: https://github.com/palantir/blueprint/blob/bcdb2bceb3bfd96252db07bda4d26fb302f4523b/packages/datetime/src/shortcuts.tsx#L119
